packages/app/src/components/routes.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { useEffect } from 'react';
+import { Home } from '../pages/home';
+import { Create, SaveKey } from '../pages/sign-up';
+import { SignIn, DecryptRecoveryCode } from '../pages/sign-in';
+
+import { Username } from '../pages/username';
+import { UsernameRegistryError } from '../pages/registery-error';
+import { SecretKey } from '../pages/secret-key';
+
+import { ChooseAccount } from '../pages/connect';
+
+import { doSaveAuthRequest } from '@store/onboarding/actions';
+import { useDispatch } from 'react-redux';
+import { ScreenPaths } from '@store/onboarding/types';
+import { doFinishSignIn as finishSignIn } from '@store/onboarding/actions';
+import { authenticationInit } from '@common/utils';
+import { useAnalytics } from '@common/hooks/use-analytics';
+import { useWallet } from '@common/hooks/use-wallet';
+import { useOnboardingState } from '@common/hooks/use-onboarding-state';
+// eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+// @ts-ignore
```

Comment:
Why's this needed?

Reply:
Because the types don't exist yet for useNavigate/Navigate in react-router-dom 6

Reply:
Added a `declare module` for this 👍